libbeat/kibana/testdata/fields.yml

```diff
@@ -0,0 +1,21 @@
+- key: test
```

Comment:
See my comment above. I suggest to use one of the `fields.yml` from a Beat, maybe Metricbeat?

Reply:
I like to have a simple test file for the tests. 

If we use a field.yml also from an external beats for more extensive tests, we should copy it here and not link it from metricbeat as there should not be dependencies from libbeat to a beat.

Reply:
I copied the file from metricbeat and checked that the new generator creates the exact same output. 
Since in the next PR I want to add bugfixes and logic enhancement I'd remove this complex test file again and make a smaller more explicit test. But for this PR I think it is a great idea to have a complex test file ensuring I didn't bring in any unintended behavior.